python/mcap/mcap0/data_stream.py

```diff
@@ -1,18 +1,15 @@
 import struct
-from io import BufferedIOBase, BytesIO
+from io import IOBase, BytesIO
 
 from .exceptions import EndOfFile
 from .opcode import Opcode
 
 
 class ReadDataStream:
-    def __init__(self, stream: BufferedIOBase):
+    def __init__(self, stream: IOBase):
```

Comment:
as far as I could tell nothing in the ReadDataStream implementation depends on buffering.